src/components/common/Footer/styles.js

```diff
@@ -1,9 +1,60 @@
-export default () => ({
+export default theme => ({
   root: {
     padding: 20
   },
+  contact: {
+    position: 'relative',
```

Comment:
Avoid using absolute/relative positioning. Use flexbox instead. If you see you really can't do it without it, then leave it as it is.

Reply:
this helps to compensate the spaces that the drops irregularly generate